### DISCUSSION TITLE
Does using Fastlane for firebase_app_distribution circumvent the need for having Firebase Distribution beta in in SPM?

### DISCUSSION THREAD

**Discussion by aj2109** [firebase/firebase-ios-sdk#12454] [Q&A] [answered]
Hi sorry for the rookie question in advance.  But when a project uses Fastlane to distribute the app using Firebase, is this under the hood actually using Firebase App Distribution installed from SPM/pods (in our case SPM), or are these entirely unrelated.  I ask because if they are unrelated we can remove it from our app and decrease the total app size.

**Comment by paulb777** [comment, marked as answer]
fastlane does not rely upon Firebase App Distribution.

**Reply by aj2109** [reply]
Thank you!
